components/theme/icons/screen.tsx

```diff
@@ -0,0 +1,46 @@
+/*
+ *
+ * Copyright (c) 2022 The Ontario Institute for Cancer Research. All rights reserved
```

Comment:
copy pasta. nom nom nom
```suggestion
 * Copyright (c) 2024 The Ontario Institute for Cancer Research. All rights reserved
```